documentation/bolt_project_directories.md

```diff
@@ -1,114 +1,89 @@
+  a `bolt-project.yaml` file to it and configuring the `modulepath` to match the
+  `modulepath` in `environment.conf`.
 
-Bolt treats a directory containing a subdirectory called `Boltdir` as a project
-directory. Use this type of directory to embed Bolt management code into another
-repo.
+## Embedded project directories
 
-For example, you can store management code in the same repo as the application
-it manages without cluttering up the top level with multiple files. This
-structure allows you to run Bolt from anywhere in the application's directory
-structure.
+Bolt treats a directory containing a subdirectory named `Boltdir` as an embedded
+project directory. Use this type of directory to embed Bolt management code into
+another repo.
 
-A project with an embedded project directory has a structure like:
+For example, you can store Bolt management code in the same repo as the
+application that Bolt manages, without cluttering up the top level with multiple
+files. This structure allows you to run Bolt from anywhere in the application's
+directory structure.
+
+An embedded Bolt directory looks like this:
 
 ```console
 project/
 ├── Boltdir
-│   ├── Puppetfile
-│   ├── bolt.yaml
 │   ├── bolt-project.yaml
-│   ├── data
-│   │   └── common.yaml
 │   ├── inventory.yaml
-│   └── site-modules
-│       └── project
-│           ├── manifests
-│           │   └── my_class.pp
-│           ├── plans
-│           │   ├── deploy.pp
-│           │   └── diagnose.pp
-│           └── tasks
-│               ├── init.json
-│               └── init.py
+│   ├── plans
+│   │   ├── deploy.pp
+│   │   └── diagnose.pp
+│   └── tasks
+│       ├── init.json
+│       └── init.py
 ├── src #non Bolt source code for the project
 └── tests #non Bolt tests for the project
 ```
 
-> **Note:** If a directory contains both `Boltdir` and `bolt.yaml` or
-  `bolt-project.yaml`, the `Boltdir` directory is used as the project directory
-  rather then the parent.
-
-### User project directory
+As long as your `bolt-project.yaml` file contains a `name` field, Bolt loads
+your local Bolt content from the top level of the `Boltdir`. Your `Boltdir` can
+also contain modules. For more information, see [Modules](modules.md).
 
-If Bolt can't find a project directory based on `Boltdir` or `bolt.yaml`, it
-uses `~/.puppetlabs/bolt` as the project directory. Use this type of directory
-if you have a single set of Bolt code and data that you use across all projects.
+## How Bolt chooses a Bolt directory
 
-## How the project directory is chosen
+If Bolt can't find a directory based on `Boltdir` or `bolt-project.yaml`, it
+uses the default: `~/.puppetlabs/bolt-project.yaml`.
 
-Bolt uses these methods, in order, to choose a project directory.
+Bolt uses the following methods, in order, to choose a Bolt directory.
 
-1. **Environment variable:** You can specify a path to a project using the
-   `BOLT_PROJECT` environment variable.
```

Comment:
```
1. **Environment variable:** You can specify a path to a project using the
   `BOLT_PROJECT` environment variable.
```
This should be added back in.

Reply:
Ah good catch - I think I missed this when I was resolving a conflict